pkg/Microsoft.Private.CoreFx.UAP/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      uap-Windows_NT;
-      uapaot-Windows_NT;
+      uap;
```

Comment:
Per above, I use the OSGroup agnostic configurations because OSGroup implies a PackageTargetRuntime.